site/docs/install-windows.md

```diff
@@ -17,40 +17,28 @@ title: Installing Bazel on Windows
 
 ### Getting Bazel
 
-Download Bazel for Windows from our
-[GitHub releases page](https://github.com/bazelbuild/bazel/releases).
-Look for `bazel-<version>-windows-x86_64.exe`, e.g. `bazel-0.13.0-windows-x86_64.exe`.
+[Download Bazel for Windows from GitHub](https://github.com/bazelbuild/bazel/releases).
+Look for `bazel-<version>-windows-x86_64.exe`, e.g. `bazel-0.15.2-windows-x86_64.exe`.
 
-**Tip:** For convenience, rename the downloaded binary to `bazel.exe` and move it to a directory
-that's on your `%PATH%` or add its directory to your `%PATH%`. This way you can run Bazel by
-typing `bazel` in any directory, without typing out the full path.
-
-**Note:** Bazel includes an embedded JDK so you don't need to install one separately. If you want
-to download Bazel without an embedded JDK, look for `bazel-<version>-without-jdk-windows-x86_64.exe`
-on the release page. To use it, you must install JDK 8 separately (older or newer versions are not
-supported), and set the `JAVA_HOME` environment variable, e.g.:
-
-    set JAVA_HOME=c:\Program Files\Java\jdk1.8.0_171
-
-Pay attention not to use quotes (") around the path (like you would on Unixes).
+**Tip:** Rename the binary to `bazel.exe` and move it to a directory on your
+`%PATH%`, so you can run Bazel by typing `bazel` in any directory.
 
 ### Other ways to get Bazel
 
-You can also get Bazel by:
-
-*   [Installing Bazel using Chocolatey](#install-using-chocolatey)
-*   [Compiling Bazel from source](install-compile-source.html)
+*   [Install Bazel using the Chocolatey package manager](#install-using-chocolatey)
```

Comment:
These bullets don't agree grammatically with the opening sentence.

"You can also get Bazel by installing/compiling, not by install/compile."

Reply:
I removed the "You can also get Bazel by:" line because it's redundant; it repeats the section header.